packages/builders/scripts/smart-wallet/build-wallet-factory2-upgrade.js

```diff
@@ -0,0 +1,29 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
```

Comment:
I don't see a `builders` package on the branch that this is going into.
I think this needs to be somewhere with a `package.json` to work.

Reply:
It's a duplicate of one in vats anyway.  Gone.